src/riak_kv_ts_error_msgs.hrl

```diff
@@ -87,3 +87,9 @@
         ["The '", ParamName, "' parameter is part the primary key, and must have an ",
          "equals clause in the query but the ", atom_to_list(Op), " operator was used."])
 ).
+
+-define(
+   E_TOO_MANY_SUBQUERIES(N),
```

Comment:
I'm following the local conventions here (alternatively, following the path of least resistance).